deeprvat/annotations/annotations.py

```diff
@@ -1773,11 +1766,13 @@ def process_vep(
     Example:
     vep_file = process_vep(vep_file, vepcols_to_retain=["additional_col1", "additional_col2"])
     """
-    vcf_df = pd.read_table(
-        vcf_file, names=["chrom", "pos", "#Uploaded_variation", "ref", "alt"]
-    )
     if "#Uploaded_variation" in vep_file.columns:
-        vep_file = vep_file.merge(vcf_df, on="#Uploaded_variation")
+        vep_file[["chrom", "pos", "ref", "alt"]] = (
+            vep_file["#Uploaded_variation"]
+            .str.replace("_", ":")
+            .str.replace("/", ":")
+            .str.split(":", expand=True)
+        )
```

Comment:
#Uploaded_variation refers to the ID column in the vcf files, these IDs can be anything (int, string, or chrom/pos/alt/ref values). So relying on these columns to contain chrom, pos, ref and alt information will fail if the IDs are e.g. int values